### DISCUSSION TITLE
are scraping / mirror websites within scope?

### DISCUSSION THREAD

**Discussion by ghost** [finnish-easylist-addition/finnish-easylist-addition#444] [General]
I'm referring to sites which exclusively or entirely scrape content from elsewhere, generally without an attribution — it'd be nice to a have repository to report Finnish sites.

However, I'm not aware if any large filter lists include such domains on their main blocklist.

for instance, suomisanakirja.fi scrapes kielitoimistonsanakirja.fi

```
https://web.archive.org/web/20230630072605/https://www.suomisanakirja.fi/is%C3%A4  
https://www.kielitoimistonsanakirja.fi/#/is%C3%A4?searchMode=all
```
see this developer-orientated uBlock Origin filter list: https://github.com/quenhus/uBlock-Origin-dev-filter/

---

**UPDATE**:

The commercial `suomisanakirja.fi` also doesn't credit the original source (which I forgot to mention). I contacted Kotimaisten kielten keskus / The Institute for the Languages of Finland (the maintainer of `kielitoimistonsanakirja.fi`); they deemed the practice as 'unethical' which they haven't been able to stop. 

The website is also seemingly SEO-optimized, [gets frequently shared on the IT-focused TechBBS forum](https://bbs.io-tech.fi/search/2338206/?q=suomisanakirja.fi&o=date) for instance.

**Comment by peace2000** [comment]
I don't see why that kind of sites would not be supported in adblocking sense as long as they are Finnish sites.

-----------------------------
Btw, I didn't know this:
`suomisanakirja.fi scrapes kielitoimistonsanakirja.fi`

**Comment by ghost** [comment]
@peace2000 I was wondering if you'd be open to **blocking** the mirror / scraper domains altogether, I wasn't referring to site-specific adblocking.

I don't know if a general filter list can expand its scope — to some blocking the domains on the main [`Finland_adb.txt`](https://github.com/finnish-easylist-addition/finnish-easylist-addition/blob/master/Finland_adb.txt) list may be comparable to censorship.

I don't even necessarily suggest blocking such domains, just interested to hear your thoughts about the subject.

**Reply by peace2000** [reply]
>@peace2000 I was wondering if you'd be open to blocking the mirror / scraper domains altogether, I wasn't referring to site-specific adblocking.

Blocking such domains is not within the scope of the Finnish adblocking list. For blocking that kind of domains, a separate list would be needed. Such as the one you linked https://github.com/quenhus/uBlock-Origin-dev-filter/. You can contribute and send those copycat domains to that list and help to make it better in fulfilling it's stated purpose.

The Finnish list is being used as a default list in many adblockers, I don't think that they expect (or want) this list to suddenly start blocking that kind of sites.

>I don't know if a general filter list can expand its scope — to some blocking the domains on the main [Finland_adb.txt](https://github.com/finnish-easylist-addition/finnish-easylist-addition/blob/master/Finland_adb.txt) list may be comparable to censorship.

Yeah, that's the case. In the past, the previous Finnish list maintainer went political https://github.com/uBlockOrigin/uBlock-issues/issues/285 - needless to say, it caused uproar. Don't know whether blocking copycat sites is as severe as political blocking, but anyway I think those domains don't belong to the Finnish list.

**Comment by Charlotte-br560** [comment]
Scraping/mirror websites fall within the scope if they meet the criteria of unauthorized content duplication without proper attribution. Reporting Finnish sites like suomisanakirja.fi would contribute to maintaining integrity online. However, checking if large filter lists include such domains is crucial to ensure effective action. The developer-oriented uBlock Origin filter list you gave me seems relevant for addressing this issue. Thanks for sharing!
